### PR TITLE
Add Beetle ESP3-C3

### PR DESCRIPTION
**Pull request by askpatrickw**

### CODE REVIEW

**Comment by askpatrickw** [comment]
This should probably go after https://github.com/adafruit/circuitpython/pull/6615 is merged

File: _board/beetle-esp32-c3.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+manufacturer: "DFRobot"
+board_url: "https://wiki.dfrobot.com/SKU_DFR0868_Beetle_ESP32_C3"
+board_image: "beetle-esp32-c3.png"
+date_added: 2022-07-17
+downloads_display: true
+blinka: false
+family: esp32c3
+bootloader_id: lolin_c3_mini
+features:
+  - Arduino Shield Compatible
```
Comment (makermelissa) [inline _board/beetle-esp32-c3.md:15]: Arduino Shield Compatible means the board is shaped like https://www.adafruit.com/product/2488 and a shield like https://www.adafruit.com/product/94 would fit on top.

**Review by makermelissa — CHANGES_REQUESTED** [review]
Thanks for adding. Most of this looks good, but it does not appear to be Arduino Shield Compatible.

File: _board/beetle-esp32-c3.md (hunk 1 above)
Comment (askpatrickw) [reply _board/beetle-esp32-c3.md:15]:
```suggestion
```

Thanks for catching that. I'm not Arduino knowledgeable and I had meant to look into that to verify.

**Review by makermelissa — APPROVED** [review]
Looks good. Now just waiting on https://github.com/adafruit/circuitpython/pull/6615

**Comment by makermelissa** [comment]
Looks like the PR this was waiting on has now been merged.

**Comment by askpatrickw** [comment]
Thank you @makermelissa !

**Pull request merged**